src/JSOSuite.jl

```diff
@@ -163,4 +163,37 @@ end
 
 include("solve.jl")
 
+"""
+    stats = feasible(nlp::Union{AbstractNLPModel, JuMP.Model}; kwargs...)
+    stats = feasible(nlp::Union{AbstractNLPModel, JuMP.Model}, solver_name::Symbol; kwargs...)
+
+Compute a feasible point of the optimization problem `nlp`. The signature is the same as the function [`solve`](@ref).
```

Comment:
Then maybe call this method `feasible_point()`?

Reply:
Yes, that's much better. Thanks for the suggestion.